lib/src/test/kotlin/starknet/provider/ProviderTest.kt

```diff
@@ -185,6 +175,17 @@ class ProviderTest {
         assertNotNull(response)
     }
 
+    @Test
+    fun `get class definition at latest block`() {
+        // FIXME: Devnet only support's calls with block_id of the latest or pending. Other block_id are not supported.
+        // After it's fixed add tests with 1) block hash 2) block number
```

Comment:
This was partially addressed in devnet I believe, that is there is functionality to make calls with blockHash and blockNumber if they are of latest/pending block.

Though I think some of the methods still don't support it, we could look into that.

Reply:
if I remember correctly I tried it that way, and it didn't work. I will create an issue to investigate it properly, and maybe fix in devnet

https://github.com/software-mansion/starknet-jvm/issues/215